sova/git/rebase.py

```
@@ -32,6 +178,41 @@ async def _get_conflicted_files(cwd: Path | None = None) -> list[str]:
+        from sova.llm.litellm_provider import _HAS_LITELLM, LiteLLMProvider
+
+        if not _HAS_LITELLM:
+            log.warning(
+                "git.rebase.litellm_unavailable",
+                msg="Multi-model conflict resolution requires litellm. Falling back to single-model.",
+            )
+            return None
+        providers: dict[str, LLMProvider] = {}
+        for model_id in models:
+            providers[model_id] = LiteLLMProvider(model=model_id)
+        return providers
```

Ruff (0.16.1)

[warning] Consider moving this statement to an `else` block

(TRY300)